includes/wikia/services/WikiService.class.php

```diff
@@ -47,66 +52,74 @@ class WikiService extends WikiaModel {
 	 * @return array of $userIds
 	 */
 	public function getWikiAdminIds( $wikiId = 0, $useMaster = false, $excludeBots = false, $limit = null, $includeFounder = true ) {
```

Comment:
Did some clean up here.  There were three levels of nested "Is this condition true?  If not, stop doing anything"  For those we might as well bail immediately and not create so much nesting and indenting.